tools/scoring/score/dimensions/dimension.py

```diff
@@ -337,14 +337,14 @@ def match_virtual_entities(
         none_subscore_reference = _VirtualEntityMatch(
             correct=0,
             correct_ceiling=len(solution_raw_field_names),
-            incorrect=0,
+            incorrect=len(solution_raw_field_names),
             proposed=set([]),
             solution=solution_parameters,
             types_correct=0,
             types_correct_ceiling=len(
```

Comment:
same here. thats a bit odd to have two variables with the same values.

Reply:
As above.